File: programmers/lev1/76501.py
def solution(absolutes, signs):
    answer = 0

    for i in range(len(absolutes)):
        if signs[i]:
            answer+=absolutes[i]
        else:
            answer+=(-absolutes[i])
    return answer

absolutes=[4,7,12]
signs=[true,false,true]
solution(absolutes, signs)

File: java/src/programmers/year2023/lv1/_76501.java
package programmers.year2023.lv1;

public class _76501 {
    public static void main(String[] args) {
        int[] absolutes = {4,7,12};
        boolean[] signs = {true, false, true};

        solution(absolutes, signs);
    }

    public static int solution(int[] absolutes, boolean[] signs) {
        int answer = 0;

        for (int i = 0; i < absolutes.length; i++) {
            int absolute = absolutes[i];
            boolean sign = signs[i];
            if (sign) {
                answer += absolute;
            } else {
                answer -= absolute;
            }
        }

        return answer;
    }
}
